eclair-core/src/main/resources/reference.conf

```
@@ -25,8 +25,8 @@ eclair {
     rpcpassword = "bar"
     // Name of the bitcoind wallet that should be used to fund channels.
     // Once set you should NOT change it if your node has channels open, otherwise you may lose funds.
-    // NB: leave empty to automatically select the default loaded wallet.
-    wallet = ""
+    // NB: leave undefined to automatically select the default loaded wallet.
+    #wallet = ""
```

This is confusing, because we can interpret `undefined` as `set to ""`...
What about:

```suggestion
    // NB: leave this field commented to automatically select the default loaded wallet.
    // wallet = ""
```

It's also probably worth updating our `README` section that describes that field.

---

> It's also probably worth updating our README section that describes that field.

Interestingly, the README is still accurate ^^

> Eclair will use the default loaded Bitcoin Core wallet to fund any channels you choose to open.
If you want to use a different wallet from the default one, you must set `eclair.bitcoind.wallet` accordingly in your `eclair.conf`.

---

> Interestingly, the README is still accurate ^^

Nice! We really were future-proof :D